.github/workflows/ci.yaml

```diff
@@ -76,7 +92,25 @@ jobs:
       - name: deploy to production
         run: |
           queenbee
-          pollination dsl push pollination-annual-daylight -e https://api.pollination.cloud -src https://api.pollination.cloud/registries --push-dependencies
+          pollination dsl push pollination-annual-daylight --tag ${{steps.get-context.outputs.tag}} -e https://api.pollination.cloud -src https://api.pollination.cloud/registries --push-dependencies
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          QB_POLLINATION_TOKEN: ${{ secrets.POLLINATION_LADYBUGBOT_TOKEN }}
+      - uses: actions/checkout@v2
+        with:
+          ref: visualization
+          fetch-depth: 0
+      - name: rebase visualization branch
+        run: |
+            git config --global user.email "ci@pollination.cloud"
+            git config --global user.name "GitHub Actions"
+            git rebase origin/production
+      - name: install python viz dependencies
+        run: pip install .
+      - name: deploy viz to production
+        run: |
+          queenbee
+          pollination dsl push pollination-annual-daylight --tag ${{steps.get-context.outputs.tag}}-viz -e https://api.pollination.cloud -src https://api.pollination.cloud/registries --push-dependencies
```

Comment:
This is probably my lack of knowledge but do we need a step called `get-context` for this to work? Or is it a special environmental var that can be accessed as a step?

Reply:
Nope, good catch! I hadn't stolen all of Antoine's previous changes.